subsys/bluetooth/controller/hci_internal.c

```diff
@@ -33,6 +33,8 @@ static struct
 	uint8_t raw_event[BT_BUF_EVT_RX_SIZE];
 } cmd_complete_or_status;
 
+static hci_internal_user_cmd_handler_t user_cmd_handler;
```

Comment:
should there be a KConfig for this?

Reply:
I considered that. But I think that is not really needed to add a Kconfig just to save a few bytes of code